app/models/reservation_validations.rb

```diff
@@ -45,6 +45,23 @@ def available
       "full time period requested.\n")
   end
 
+  def check_consecutive
+    return unless reserved? || requested?
+    return unless equipment_model
+    return unless equipment_model.max_checkout_length
+
+    # get all reservations of same user with same equipment model
+    # that are consecutive to this reservation
+    res = Reservation.for_reserver(reserver).for_eq_model(equipment_model)
+          .consecutive_with(self)
+
+    res.each do |r|
+      next unless duration + r.duration > equipment_model.max_checkout_length
```

Comment:
Oh man, that's a great point (having multiple of a given equipment model). I'll e-mail BMEC to check and see what they want to do because it gets a little tricky between having both a max per user and trying to handle this case. Nice catch!